javascript/chronometer.js

```diff
@@ -1,26 +1,39 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = 0;
   }
+
   startClick(callback) {
-    // ... your code goes here
+    this.intervalId = setInterval(() => {
+    this.currentTime += 1
```

Comment:
after this line you will want to execute the callback function.
`callback()`
This function will be the printTime function from index.js, just passed as a parameter. And you want to run it every second, so, inside the interval.